src/main/java/kg/jarkyn/cobspecserver/Runner.java

```diff
@@ -12,55 +12,64 @@
 
 import java.io.IOException;
 import java.net.ServerSocket;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
 
 public class Runner {
-    private static boolean KEEP_RUNNING = true;
-    private static int defaultPort      = 5000;
-    private static String publicResourcePath = "/Users/Jarkyn/Projects/8thLight/JAVA/cob_spec/public";
+    private static final int DEFAULT_PORT = 5000;
+    private static final boolean KEEP_RUNNING = true;
+    private static final String DOMAIN = "http://localhost";
+    private static final String PUBLIC_RESOURCE_PATH = "/Users/Jarkyn/Projects/8thLight/JAVA/cob_spec/public";
```

Comment:
:) someone else suggested that they should be, they weren't before.
It was not a requirement for me to parse args, I have a very clear idea how I would do it when given the time.